int32 MOTION_TO_GOAL=           1
int32 MOTION_ODO_CALIB=         2
int32 MOTION_POTENTIAL_FIELD=   3
#goal definition
int32 mode
float32 beta
float32 x
float32 y
float32 theta
float32 v
int32 servof
int32 servor
float32 deltaf
float32 deltar
---
#result definition
int32 MOTION_STATUS_OK=         0
int32 MOTION_STATUS_COLLISION=  1
int32 mode
int32 status
float32 dist_driven
float32 dist_goal
---
#feedback
int32 MOTION_STATUS_OK=         0
int32 MOTION_CALIB_OK=         0
int32 mode
float32 dist_driven
float32 dist_goal
